src/components/ComponentPages/DelegatedSupportCamps/__test__/delegatedSupportCamps.test.tsx

```diff
@@ -134,7 +134,7 @@ describe("Delegated Support camps page", () => {
     );
     expect(getByText("Remove")).toBeTruthy();
     expect(getByText("Cancel")).toBeTruthy();
-    expect(screen.getByText("Remove Support")).toBeInTheDocument();
+    expect(screen.getAllByText("Remove Support")).toBeTruthy();
```

Comment:
Please move "Remove Support" to common message or label file

Reply:
Done